code/game/turfs/simulated/dirtystation.dm

```diff
@@ -137,11 +117,7 @@
 
 	if(istype(A, /area/toxins))
 		if(prob(80))
-			new /obj/effect/decal/cleanable/dirt(src)
-		else
 			new /obj/effect/decal/cleanable/greenglow(src)	//this cleans itself up but it might startle you when you see it.

```

Comment:
Eh, the way you removed the dirt statement here and above in the QM statement... 

Before it would only make greenglow 20% of the time. Now it'll make it 80% of the time. This might make the QM a bit too oily.